examples/github-lfs/compose.yaml

```diff
@@ -0,0 +1,162 @@
+name: github-lfs
+
+volumes:
+  lfs-storage: {}
+
+services:
+  giftless:
+    image: docker.io/datopian/giftless:latest
```

Comment:
We might actually want to point to the lsst-sqre one for now.  I haven't been making any headway on getting access to PyPi and Docker from Datopian.  And honestly I don't want write access to their repositories, but I would like to put newer versions out there.

Reply:
Well, that's a [Datopian] shame, but in this example it's just a local name (as the pull policy is cleverly set to `Never` :rofl: )